### ISSUE TITLE
Pandas FutureWarning on JSON Normalize

### ISSUE THREAD

**Issue by jcherry-risklens** [RiskLens/verispy#28]
Woo fun, new byproduct of the v1 Pandas library I imagine:
```
verispy/veris.py:77: FutureWarning: pandas.io.json.json_normalize is deprecated, use pandas.json_normalize instead
  df_comb = pd.io.json.json_normalize(jsons)
```
Hopefully should be a simple line change.

**Comment by tbyers-risklens** [comment]
I haven't upgraded pd yet...please let me know what other version > 1.0 issues you see

**Comment by tbyers-risklens** [comment]
fixed in 93d29b46c8aaff082405f9884e6f4eade29c7af3
